types/configurable/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for configurable 0.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Vilim Stubičan <https://github.com/jewbre>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+// Make any object configurable
+export default function(obj: any): Configurable;
```

Comment:
and seems this function mixes some declarations in, so it should be somethign like:

```ts
function configurable<T extends object>(obj: T):  T & Configurable; 
export = configurable;
```

Reply:
do not export `default` since the runtime npm package does not have a default export. see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
You are completely right. 

I have additionally added a generic to the class since `configurable` mixins return `this`.
Also, removed `default` export.